draft-ietf-rats-corim.md

```diff
@@ -87,6 +87,7 @@ normative:
     target: https://www.itu.int/rec/T-REC-X.690
   IANA.named-information: named-info
 
+
```

Comment:
If we add that reference, I'd recommend to add below line 59.
Where will that reference be used in text?

Reply:
I assume it would be informative (not sure what "below line 59" means). A reference to a pkix- thing might cite it or not based on whether or not it is a normative reference correct?